packages/bitcore-lib/lib/address.js

```diff
@@ -488,6 +545,11 @@ Address.prototype.toObject = Address.prototype.toJSON = function toObject() {
  * @returns {string} Bitcoin address
  */
 Address.prototype.toString = function() {
+  if (this.isPayToWitnessPublicKeyHash() || this.isPayToWitnessScriptHash()) {
+    var prefix = this.network['bech32prefix'];
```

Comment:
any reason for not doing `this.network.bech32prefix`

Reply:
Fixed.